src/aap_eda/core/models/event_stream.py

```diff
@@ -0,0 +1,92 @@
+#  Copyright 2022 Red Hat, Inc.
```

Comment:
@Alex-Izquierdo should this be 2024?

Reply:
should it be the year when the application was licensed? I'm not sure. Otherwise we should update the year in all the places, don't?

Reply:
changed